modulegen/internal/model/cmd.go

```diff
@@ -0,0 +1,7 @@
+package model
```

Comment:
I'm thinking about naming this package context instead of model, not sure about it. It could probably come with some renames: `model.GetRootContext()` >> `context.New()`

I'm fine having the discussion here and implement it probably in a follow-up?

Reply:
done